src/Core_Command.php

```diff
@@ -105,7 +105,7 @@ public function check_update( $_, $assoc_args ) {
 	 * : Select which language you want to download.
 	 *
```

Comment:
I think reusing the `--version=<version>` option here is confusing, and there's no need for that in terms of back-compat.

I'd prefer for the command to just accept a URL as an optional positional argument.
```suggestion
	 *
	 * [<download-url>]
	 * : Download directly from a provided URL instead of fetching the URL from the wordpress.org servers.
	 *
```

Reply:
This will also be more in line with other commands that accept URLs.

So, just as you can do `wp plugin install https://example.com/my-plugin.zip` you will be able to do `wp core download https://example.com/my-core.zip`.

Reply:
Sounds good to me. I'll make the other needed updates tomorrow.